graphene/types/schema.py

```diff
@@ -391,6 +411,23 @@ def resolve_type(self, resolve_type_func, type_name, root, info, _type):
         return_type = self[type_name]
         return default_type_resolver(root, info, return_type)
 
+    def add_prefix_to_type_name(self, name):
+        if self.type_name_prefix:
+            return self.type_name_prefix[0].upper() + self.type_name_prefix[1:] + name
```

Comment:
```suggestion
            return self.type_name_prefix.capitalize() + name
```

Reply:
I did that at first actually, but:

<img width="230" alt="image" src="https://user-images.githubusercontent.com/26854828/197784364-092c993c-3f4a-4063-9941-786bc4645a6c.png">

Reply:
mmmm that's a fair point! (that also means that if we have auto camel case on and we have `my_fieldIsGreat = Field(...)` it would become `myFieldisgreat` because it does use `.capitalize()`... should we change that @erikwrede in your opinion?)